raiden/utils/ethereum_clients.py

```diff
@@ -30,11 +30,12 @@ def support_check(
     highest_supported_version_string: str,
     lowest_supported_version_string: str,
 ) -> bool:
-    if len(our_version._key[1]) != 3:
-        # This can happen for versions such as 0.0.0
-        return False
+    if len(our_version._key[1]) < 2:
```

Comment:
I couldn't find the documentation, but looking through the code I found that `Version` has an attribute `_version` which is a namedtuple, so this could be `our_version._version.release`, could you try that instead? (`_key` is filtered, `_version` is not, additionally the named argument is more understandable than the constants `1`)

Reply:
on a side node, why are we using a private member of an external library? isn't there a better solution?

Reply:
>  could you try that instead? 

Sure, will make this easier to read.

>  why are we using a private member of an external library? isn't there a better solution?

Not that I could find of, using `pkg_resources`. I also just found this `_key` thing through looking at the code. No luck from their docs, which is strange.

If anyone has any better, documented solution for this I am all for it.